bodhi/server/schemas.py

```diff
@@ -395,6 +395,10 @@ class SaveReleaseSchema(CSRFProtectedSchema, colander.MappingSchema):
         colander.String(),
         missing=None,
     )
+    mail_template = colander.SchemaNode(
+        colander.String(),
+        missing="fedora_errata_template",
+    )
```

Comment:
We should validate that the referenced template exists, otherwise users could make a typo and cause the server to fail during mashing:

```
[vagrant@bodhi-dev bodhi]$ bodhi releases edit --name F28 --mail-template does_not_exist                                                                   
Saved release:
  Name:                F28
  Long Name:           Fedora 28
  Version:             28
  Branch:              f28
  ID Prefix:           FEDORA
  Dist Tag:            f28
  Stable Tag:          f28-updates
  Testing Tag:         f28-updates-testing
  Candidate Tag:       f28-updates-candidate
  Pending Signing Tag: f28-signing-pending
  Pending Testing Tag: f28-updates-testing-pending
  Pending Stable Tag:  f28-updates-pending
  Override Tag:        f28-override
  State:               current
  Email Template:      does_not_exist
```